tests/drivers/sdp_asm/CMakeLists.txt

```diff
@@ -7,15 +7,9 @@
 cmake_minimum_required(VERSION 3.20.0)
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project(hello_world)
+project(sdp_asm)
 
-sdp_assembly_generate("${CMAKE_CURRENT_SOURCE_DIR}/src/add_1.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_10.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_100.c")
-sdp_assembly_check("${CMAKE_CURRENT_SOURCE_DIR}/src/add_1.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_10.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_100.c")
-sdp_assembly_prepare_install("${CMAKE_CURRENT_SOURCE_DIR}/src/add_1.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_10.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_100.c")
+sdp_assembly_install(app
+  "${CMAKE_CURRENT_SOURCE_DIR}/src/add_1.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_10.c;${CMAKE_CURRENT_SOURCE_DIR}/src/add_100.c")
```

Comment:
Please line wrap, or perhaps create a list of files which are passed as argument, like this:

```
set(sdp_files
    ${CMAKE_CURRENT_SOURCE_DIR}/src/add_1.c
    ${CMAKE_CURRENT_SOURCE_DIR}/src/add_10.c
    ${CMAKE_CURRENT_SOURCE_DIR}/src/add_100.c
)
sdp_assembly_install(app "${sdp_files}")
```